src/RelationshipClass.ts

```diff
@@ -43,7 +43,11 @@ export enum RelationshipClass {
   EXTENDS = 'EXTENDS',
 
   /**
-   * A relationship between any Entity and {Finding, Incident}.
+   * A relationship indicating an Entity is responsible for generating another Entity.
+   *
+   * Examples:
+   *
+   * * Project - GENERATED -> Finding
```

Comment:
Just so we don't lose any existing documentation

```suggestion
   * * Project - GENERATED -> {Finding, Incident}
```

Reply:
I need to find an example of that relationship, couldn't see one in GitHub, perhaps my search-fu is off in all the context switching 😬